main.go

```diff
@@ -178,7 +193,13 @@ func listCmd() *cobra.Command {
 				if onlyName {
 					fmt.Fprintln(w, np.Name)
 				} else {
-					fmt.Fprintf(w, "%s\t%5d\t%s\n", np.Name, np.Nodes, np.Type)
+					typeList := make([]string, len(np.Types))
+					i := 0
+					for k := range np.Types {
+						typeList[i] = k
+						i++
+					}
```

Comment:
Let's better use `append`, it's more _Goish_ that way; while at it let's also sort the list of types:
```suggestion
					for k := range np.Types {
						typeList = append(typeList, k)
					}
					sort.Strings(typeList)
```

Reply:
I have committed your suggestion as I think it's easier to read and performance doesn't matter here

Reply:
> I think since we know the size of this array ahead of time,[ it's more efficient to set the value explicitly](https://stackoverflow.com/questions/38654729/golang-slice-append-vs-assign-performance) rather than dynamically re-sizing the slice.

I've finally had the time to come back to this. That SO answer is a bit misleading: while it's true dynamically re-sizing the slice is less performant that is only true if you don't create the slice with the right cap. As you can see in [this benchmark](https://gist.github.com/inkel/a79f6e27b4b2cc7a4bc3ccdd182dbaa3) if you create the slice with 0 length but the right capacity it is almost as fast as using an explicit index:

```
BenchmarkSetSpeed/explicit_index-12         	10000000	      177.5 ns/op	    160 B/op	      1 allocs/op
BenchmarkSetSpeed/append_len=0_cap=len(set)-12         	10000000	      180.5 ns/op	    160 B/op	      1 allocs/op
BenchmarkSetSpeed/append_len=0_without_cap-12          	10000000	      427.7 ns/op	    496 B/op	      5 allocs/op
```